crmsh/bootstrap.py

```diff
@@ -186,6 +190,19 @@ def init_sbd_manager(self):
         from .sbd import SBDManager
         self.sbd_manager = SBDManager(self)
 
+    def load_profiles(self):
+        """
+        Load profiles data for different cloud provider
+        """
+        if not os.path.exists(PROFILES_FILE):
+            return
+        env_name = utils.detect_cloud()
+        if not env_name:
+            env_name = USER_CUSTOMIZED
```

Comment:
revisit this today, I think the correct and better logic is to load  the USER_CUSTOMIZED profile as the defaults and let it overriden by the correspondig profile environment if detected.

Reply:
It's the same here
If detected one of three could providers, then use that profiles data; else, use the default one

Reply:
Okay, fine, it can work. It depends on how we interpret the usage of profiles.yml indeed.